src/components/notebook/macro-summation.test.tsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import { MacroSummation } from "./macro-summation";
+import { TrayObject, AnimalType } from "../../utils/sim-utils";
+import { getPTIScore } from "../../utils/macro-utils";
+
+const trayObjectsExcellent: TrayObject[] = [
+  { type: AnimalType.aquaticWorm, count: 4, collected: true, image: "", selectionPath: "", zIndex: 0,
+    trayIndex: 0, left: 0, top: 0, width: 0, height: 0, boundingBoxHeight: 0, boundingBoxWidth: 0, rotation: 0 },
```

Comment:
It'd be nice to do something about the redundancy here. Perhaps something like:
```typescript
const testTrayObject = (type: AnimalType, count: number): TrayObject => {
  return { type, count, ...other defaulted properties... };
};

const trayObjectsExcellent = [
  testTrayObject(AnimalType.aquaticWorm, 4),
  testTrayObject(AnimalType.caddisFly, 9),
  ...
];
```

Reply:
Fixed.